README.md

```diff
@@ -38,45 +40,91 @@ This project is inspired by [Google vizier](https://static.googleusercontent.com
 
 Katib stands for `secretary` in Arabic. As `Vizier` stands for a high official or a prime minister in Arabic, this project Katib is named in the honor of Vizier.
 
-## Concepts in Google Vizier
+## Concepts in Katib
 
-As in Google Vizier, Katib also has the concepts of Study, Trial and Suggestion.
+As in [Google Vizier][3], Katib also has the concepts of Experiment, Trial and Suggestion.
```

Comment:
Do we need to say Vizier in the doc? The terminologies have diverged in v1alpha2 and design has been changed

Reply:
With Suggestion CRD(next release), there is hardly any similarity with vizier

Reply:
I think we could say it is inspired by vizier and do not introduce vizier in the doc, WDYT.

Reply:
LGTM. I feel that we do not need to even mention 'vizier' when v1alpha1 is deprecated.

Reply:
I think our concepts about trial and study is inspired by vizier :thinking: